homeassistant/components/home_connect/api.py

```diff
@@ -110,13 +117,17 @@ class HomeConnectDevice:
     # for some devices, this is instead BSH_POWER_STANDBY
     # see https://developer.home-connect.com/docs/settings/power_state
     power_off_state = BSH_POWER_OFF
+    hass: core.HomeAssistant
+    appliance: HomeConnectAppliance
 
-    def __init__(self, hass, appliance):
+    def __init__(
+        self, hass: core.HomeAssistant, appliance: HomeConnectAppliance
```

Comment:
```suggestion
        self, hass: HomeAssistant, appliance: HomeConnectAppliance
```
Please don't import core